.github/workflows/e2e.yml

```diff
@@ -57,9 +57,16 @@ jobs:
           export E2E_DOCKER_CONFIG_JSON_SECRET=`cat ~/.docker/config.json| base64 -w 0`
           make e2e E2EARGS="-v=5 -skip-features 'pv expansion|custom config for dynamic|tools for exporter'"
         env:
+          E2E_AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          E2E_AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          E2E_BR_IMAGE: reg.vesoft-inc.com/cloud-dev/br-ent
+          E2E_BR_VERSION: v3.7.0
+          E2E_GOOGLE_APPLICATION_CREDENTIALS: ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS }}
           E2E_OPERATOR_IMAGE: reg.vesoft-inc.com/ci/nebula-operator:ci-e2e
           E2E_OPERATOR_INSTALL: "true"
-          E2E_NC_VERSION: v3.6.0
+          E2E_NC_VERSION: v3.7.0
+          E2E_NC_AGENT_IMAGE: reg.vesoft-inc.com/cloud-dev/nebula-agent
```

Comment:
Why put it in `cloud-dev`?

Reply:
@MegaByte875 to move them into the CI folder